src/main/java/com/uber/cadence/worker/WorkflowImplementationOptions.java

```diff
@@ -19,13 +19,17 @@
 
 import static com.uber.cadence.worker.NonDeterministicWorkflowPolicy.BlockWorkflow;
 
+import com.uber.cadence.activity.ActivityOptions;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Objects;
 
 public final class WorkflowImplementationOptions {
 
   public static final class Builder {
 
     private NonDeterministicWorkflowPolicy nonDeterministicWorkflowPolicy = BlockWorkflow;
+    private Map<String, ActivityOptions> activityOptions = new HashMap<>();
```

Comment:
might be worth a comment as to what the intent of the string key is here, as I understand it it's the identifier of the activity, but just from looking at the code that's not super obvious

Reply:
there is a comment that says that the key is the activity name. The comment is above setActivityOptionOverrides method. I assume it should be fine?